### DISCUSSION TITLE
[Remix] Deployment Issues on latest Vercel CLI

### DISCUSSION THREAD

**Discussion by fvaldes33** [vercel/community#2310] [Bug Report]
Deployment on Vercel show as successful but the logs are weird and the website is broken. Had to revert the CLI to 29.0.3 to get it working. 

Looks like all packages are "missing". Seeing messages like this for basically every package.

```
The path "@remix-run/react" is imported in app/entry.client.tsx but "@remix-run/react" was not found in your node_modules. Did you forget to install it?
```

Vercel CLI: 29.1.1

package.json
```
@remix-run/node: "^1.7.5",
@remix-run/react: "^1.7.5",
@remix-run/vercel: "^1.7.5",

// dev deps
@remix-run/dev: "^1.7.5",
@remix-run/eslint-config: "^1.7.5",
@remix-run/serve: "^1.7.5",
```

**Comment by roniaxe** [comment]
I'm having the same issue, not using vercel CLI so I'm not sure how to work around it.

Build logs:
https://vercel.com/roniaxe-gmailcom/wellspring-montessori-frontend/27Smnz1ddGhnBykeMg3NtgScA8ej
